test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -647,69 +639,55 @@ spec setup =
             do_round -3.50001 . should_equal -4
             do_round -3.99999 . should_equal -4
 
-        Test.specify "Returns the correct type" <|
-            do_round 231.2 1 . should_be_a Decimal
-            do_round 231.2 0 . should_be_a Decimal
-            do_round 231.2 . should_be_a Decimal
-            do_round 231.2 -1 . should_be_a Decimal
-
         Test.specify "Can round correctly near the precision limit" <|
             do_round 1.22222222225 10 . should_equal 1.2222222223
             do_round 1.222222222225 11 . should_equal 1.22222222223
             do_round 1.2222222222225 12 . should_equal 1.222222222223
             do_round 1.22222222222225 13 . should_equal 1.2222222222223
             do_round 1.222222222222225 14 . should_equal 1.22222222222223
-            do_round 1.2222222222222225 15 . should_equal 1.222222222222223
 
             do_round -1.22222222225 10 . should_equal -1.2222222223
             do_round -1.222222222225 11 . should_equal -1.22222222223
             do_round -1.2222222222225 12 . should_equal -1.222222222223
             do_round -1.22222222222225 13 . should_equal -1.2222222222223
             do_round -1.222222222222225 14 . should_equal -1.22222222222223
-            do_round -1.2222222222222225 15 . should_equal -1.222222222222223
```

Comment:
Why are these removed?

Reply:
Because SQLite fails this test. I could add this one line to the backend-specific test, with different values.

Reply:
Why does it fail it? Should we be concerned? (if not a big deal, I'm ok with removing it)

Reply:
Moved to backend-specific tests.